src/libs/PolicyUtils.ts

```diff
@@ -538,6 +539,49 @@ function canUseProvincialTaxNetSuite(subsidiaryCountry?: string) {
     return subsidiaryCountry === '_canada';
 }
 
+function getNetSuiteReimbursableAccountOptions(policy: Policy | undefined, selectedBankAccountId: string | undefined): SelectorType[] {
+    const payableAccounts = policy?.connections?.netsuite.options.data.payableList ?? [];
+    let accountsToConsider = (payableAccounts ?? []).filter(({type}) => type === CONST.NETSUITE_ACCOUNT_TYPE.BANK);
```

Comment:
```suggestion
    const accountOptions = (payableAccounts ?? []).filter(({type}) => 
    type === CONST.NETSUITE_ACCOUNT_TYPE.BANK || type === CONST.NETSUITE_ACCOUNT_TYPE.CREDIT_CARD);

```

Reply:
That's actually just an arbitrary quirk on OldDot. We don't need to follow that here.